s14/lambda.rb

```diff
@@ -0,0 +1,34 @@
+squares_proc =  Proc.new {|n| n ** 2}
+p [1, 2, 3].map(&squares_proc)
+p squares_proc.call(5)
+
+squares_lambda = lambda {|n| n ** 2}
```

Comment:
What's the difference between `Proc` and `Lambda`?

Reply:
That Lambda unlike Proc is strict in receiving the requested arguments.